compiler/rustc_middle/src/ty/region.rs

```diff
@@ -0,0 +1,3 @@
+use crate::ty::{BoundRegion, Placeholder};
```

Comment:
You should just move it into sty with the rest of the region variants, then later if you want to break up sty, all the region variants can be moved into a file here. This file in its current state is kinda misleading, lmao